README.md

```diff
@@ -94,6 +96,24 @@ You now specify your action space in the json file you pass in through
 `MODEL_METADATA_FILE_S3_KEY`, which is defaulted to
 `bucket/custom_files/model_metadata.json`
 
+### Altering hyperparameters
+To override default hyperparameters, in `rl_deepracer_coach_robomaker.py` uncomment the fields that you wish to override and set their value.
+
+Default values can be found in `sagemaker_graph_manager.py`
+
+### Using a pretrained model
```

Comment:
This has a part in the wiki as well. A PR needs to be there

Reply:
I raised #29 - I cannot submit PRs against the wiki